### PR TITLE
Add mobile layout first step

### PR DESCRIPTION
**Pull request by pierresisson**
### What does it do ? Why ?
Add a mobile layout and work as mobile first.
Meaning we have to think and write for a mobile view first and then others devices.

Example: `<div className="grid grid-cols-4 gap-6 lg:grid-cols-2">`

Steps :
- Fix global grid on mobile
- Fix aside width
- Convert `Tabs` to `Link`
image

### QA
Check if elements fit in mobile

### CODE REVIEW

**Review by cypherhoodlum — CHANGES_REQUESTED** [review]
Code looking good. What are you using to check the mobile responsiveness? Maybe a more broad css containerisation would be in order? Firefox devtools looking like this at the moment:

Screenshot from 2023-03-06 12-01-23

**Comment by pierresisson** [comment]
@cypherhoodlum looking on this

**Comment by pierresisson** [comment]
should be better now @cypherhoodlum 
image

File: src/app/[entity]/[repo]/page.tsx [hunk 1]
```diff
@@ -57,83 +62,146 @@ export default function RepoPage({
...
+                <Badge className="ml-2">209</Badge>
+              </Button>
+              <Button
+                className="h-8 !border-[#383B42] bg-[#22262C] text-xs"
+                variant="outline"
+              >
+                <Star className="mr-2 h-4 w-4 text-yellow-500" /> Starred
+                <Badge className="ml-2">7k</Badge>
+              </Button>
+            </div>
```
Comment (pierresisson) [inline src/app/[entity]/[repo]/page.tsx:139]: we can factorize this in another pr i think

Comment (PeerRich) [reply src/app/[entity]/[repo]/page.tsx:139]: yea

**Comment by pierresisson** [comment]
arf the burger pull request destroy this PR :D i need to check conflicts

**Comment by BruceWangyq** [comment]
> arf the burger pull request destroy this PR :D i need to check conflicts

> Haha. Sorry, Bro, you can try to rebase it.

**Comment by PeerRich** [comment]
too many engineers already working on it! lovely 💜

**Comment by pierresisson** [comment]
no worries @BruceWangyq ! yes im rebasing but a lot of conflicts, but it's ok, i'm on it 🤓

**Comment by pierresisson** [comment]
@BruceWangyq can you check again ? poke @PeerRich

**Comment by PeerRich** [comment]
i think the odd border is back

**Review by PeerRich — COMMENTED** [review]
just checked out locally and found some issues:

spacing between buttons missing:

CleanShot 2023-03-06 at 19 14 01@2x

divider not under tabs

CleanShot 2023-03-06 at 19 14 29@2x

can't scroll tabs (cut off):

CleanShot 2023-03-06 at 19 15 27@2x

**Comment by BruceWangyq** [comment]
> no worries @BruceWangyq ! yes im rebasing but a lot of conflicts, but it's ok, i'm on it 🤓

Yeah, the odd color happened because after I applied the tailwind config color, 
```
theme: {
    extend: {
      colors: {
        gray: "#383B42",
        dark: "#22262C",
      },
    },
  }, 
```
something like `border-gray-400/20 `is not working, you can just make it "border-gray", it is the same as `border-[#383B42]`, and it looks just like `border-gray-400/20`. That should be working fine and cleaner. @PeerRich

**Comment by BruceWangyq** [comment]
> @BruceWangyq can you check again ? poke @PeerRich

**Comment by cypherhoodlum** [comment]
Looking at this now. A gentle reminder to use issue numbers when naming branches. Maybe this is a pre-policy branch :smile:

**Comment by PeerRich** [comment]
CleanShot 2023-03-06 at 20 00 59@2x
this still on my end

**Comment by pierresisson** [comment]
@PeerRich the burger pull request changed this I guess? dunno, I will add a gap

**Comment by cypherhoodlum** [comment]
I think this one needs a proper conflict resolve session :smile: I'll try to sort it out on my end but I'm not pushing anything until I have a mobile responsive layout. Feel free to keep working on it in the meantime.

**Comment by pierresisson** [comment]
done @cypherhoodlum and @PeerRich 👌🏻

**Comment by cypherhoodlum** [comment]
I think we need a css container for the whole page.tsx which forces the width to stay within the viewport.

**Comment by pierresisson** [comment]
if we can merge this first it would be great, what do you think @cypherhoodlum ?

**Comment by cypherhoodlum** [comment]
Looking at it as soon as I resolve a merge conflict. Which one is the correct one?
Screenshot from 2023-03-06 21-12-58

**Comment by cypherhoodlum** [comment]
Let's merge this and look at the viewport issue later.

**Comment by cypherhoodlum** [comment]
@pierresisson do you feel up to merging with main before we can proceed? :smile:

**Comment by pierresisson** [comment]
@cypherhoodlum not agaaaain ahaha yes, im on it

**Comment by cypherhoodlum** [comment]
I feel your pain :smile:

**Comment by pierresisson** [comment]
this is ok 
image

**Comment by PeerRich** [comment]
i think we can merge this and do another follow up later

**Comment by pierresisson** [comment]
image

**Comment by pierresisson** [comment]
yes, lets merge

**Pull request merged**